src/styles/header/navigation.scss

```diff
@@ -0,0 +1,33 @@
+.nav {
+  display: inline-block;
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    display: inline-block;
+  }
+
+  &__link {
+    display: inline-block;
+    color: #000;
```

Comment:
Consider changing this color accordingly to the Figma design, it would help to pass all tests